README.md

```diff
@@ -3,6 +3,18 @@
 A high performance implementation of the Ruby programming language. Built on the
 GraalVM by [Oracle Labs](https://labs.oracle.com).
 
+## Aim
+
+TruffleRuby aims to:
+
+* Run idiomatic Ruby code faster
+* Run Ruby code in parallel
+* Boot Ruby applications in less time
+* Execute C extensions in a managed environment
+* Provide new tooling such as debuggers and monitoring
+* Interoperate with languages like JavaScript, Python and R
```

Comment:
Interoperate *efficiently* maybe? Or something to indicate it's much better than the traditional "patch 2 VMs together approach".

Reply:
Done.